app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionToPlaceOfWorship.java

```diff
@@ -0,0 +1,51 @@
+package de.westnordost.streetcomplete.quests.religion;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddReligionToPlaceOfWorship extends SimpleOverpassQuestType
+{
+	@Inject public AddReligionToPlaceOfWorship(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways, relations with amenity=place_of_worship and" +
+                " !religion and" +
+                " name and" +
+                " (access !~ private|no)"; // exclude ones without access to general public
```

Comment:
You'd still be able to see it from the outside.

Reply:
I thought about edge cases like PoW in a military base or a prison. Are you sure that it should be dropped?

Reply:
Yes.

Reply:
dropped